lntest/bitcoind.go

```diff
@@ -121,21 +122,32 @@ func NewBackend(miner string, netParams *chaincfg.Params) (
 		return nil, nil, fmt.Errorf("couldn't start bitcoind: %v", err)
 	}
 
-	cleanUp := func() {
+	cleanUp := func() error {
 		bitcoind.Process.Kill()
 		bitcoind.Wait()
 
+		errStr := ""
 		// After shutting down the chain backend, we'll make a copy of
 		// the log file before deleting the temporary log dir.
 		err := CopyFile("./output_bitcoind_chainbackend.log", logFile)
 		if err != nil {
-			fmt.Printf("unable to copy file: %v\n", err)
+			errStr += fmt.Sprintf("unable to copy file: %v\n", err)
 		}
 		if err = os.RemoveAll(logDir); err != nil {
-			fmt.Printf("Cannot remove dir %s: %v\n", logDir, err)
+			errStr += fmt.Sprintf(
+				"Cannot remove dir %s: %v\n", logDir, err,
```

Comment:
nit: we usually don't capitalize error strings

Reply:
fixed.